runtime/src/accounts_index.rs

```diff
@@ -1562,7 +1621,142 @@ pub mod tests {
     }
 
     #[test]
-    #[should_panic(expected = "out of range")]
+    fn test_bitfield_excess2() {
+        solana_logger::setup();
+        let width = 16;
+        let (mut bitfield, mut hash) = setup_empty(width);
+        let slot = 100;
+        // insert 1st slot
+        bitfield.insert(slot);
```

Comment:
Seems like the tests could be simplified with a kind of wrapper object with both bitfield and hashset where it has `insert` and `compare` functions. `compare` can test that bitfield.contains is tested with what exists in the reference hashset.